### PR TITLE
Feature/redis cache

### PR DESCRIPTION
**Pull request by LucasGronlund**
adds an in-memory cache. removes old storage in divs.

### CODE REVIEW

**Comment by LucasGronlund** [comment]
would be nice to have some tests, although I might do it in another PR just to get this out..

File: src/client.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import json
 import os
+import time
 from json.decoder import JSONDecodeError
+from pathlib import Path
 
 import requests
```
Comment (MackoHacko) [inline src/client.py:7]: Cleaner to move this to its friends at the top 😸

File: src/cache.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from json import dumps
```
Comment (MackoHacko) [inline src/cache.py:1]: Cleaner structure among imports plz 💛

File: src/client.py (hunk 1 above)
Comment (LucasGronlund) [reply src/client.py:7]: hmm, pretty sure i used our linters auto-structure for the imports, will check

Comment (LucasGronlund) [reply src/client.py:7]: might have forgotten it on one of the files :D

Comment (LucasGronlund) [reply src/client.py:7]: https://code.visualstudio.com/docs/python/editing#_sort-imports

File: src/cache.py (hunk 2 above)
Comment (LucasGronlund) [reply src/cache.py:1]: https://code.visualstudio.com/docs/python/editing#_sort-imports

**Comment by LucasGronlund** [comment]
open for suggestions on how to change the in-build import sorter, it's possible to use a customer script or change the settings:

https://github.com/timothycrosley/isort#configuring-isort

**Comment by LucasGronlund** [comment]
Added integration tests as well now, so don't forget to check those out :)

**Pull request merged**